sql-select/two-kilmers.sql

```diff
@@ -0,0 +1,4 @@
+select  "actorId",
+        "firstName"
+  from "actors"
+  where "lastName" = 'Kilmer'
```

Comment:
Please update this query to ensure no more than two `'Kilmer'`s are included in the results:

---

![image](https://user-images.githubusercontent.com/16143771/119073891-e3a4c900-b9a2-11eb-9f5c-fee46c81b170.png)